### DISCUSSION TITLE
The implementation of tab-transformer model

### DISCUSSION THREAD

**Discussion by linkedlist771** [autogluon/autogluon#2639] [Q&A] [answered]
Hi guys, 
           When I read the source code of the models, I found that tab-transformer code has been added to the models yet, while still not yet to the model init.py. I wonder that the tab-transformer model is ready to use or not ?

**Comment by gradientsky** [comment, marked as answer]
It is currently in experimental form and not fully tested/hardened as other production-grade models included in default configs.

**Reply by linkedlist771** [reply]
Thanks for your answer, I have just used this model morning and found that the training process takes a long time in cuda compared with the default models in CPU. And I also wonder what optimization will be added for the tab-transformer training.
